vcsetup.iss

```diff
@@ -3,44 +3,60 @@
 #define public Dependency_Path_NetCoreCheck "InnoDependencyInstaller\dependencies\"
 #include "InnoDependencyInstaller\CodeDependencies.iss"
 
-#define AppID "{33DFCEE8-022C-4C66-A366-79A7415320F2}"
-#define AppName "Volume Control"
-#define AppPublisher "radj307"
-#define AppURL "https://github.com/radj307/volume-control"
-#define AppExeName "VolumeControl.exe"
-#define AppMutex "VolumeControlSingleInstance"
+#define AppID               "{33DFCEE8-022C-4C66-A366-79A7415320F2}"
+#define AppName             "Volume Control"
+#define AppPublisher        "radj307"
+#define CurrentYear         GetDateTimeString('yyyy','','')
+#define StartYearCopyright  "2011"
+#define AppURL              "https://github.com/radj307/volume-control"
+#define AppExeName          "VolumeControl.exe"
+
+#define AppMutex            "VolumeControlSingleInstance"
 ; #define AppVersion "" ;< DEFINE THIS VIA COMMANDLINE (/dAppVersion="VERSION_NUMBER")
 
 [Setup]
-AppId={{#AppID}
+AppId={#AppID}
 AppName={#AppName}
 AppVersion={#AppVersion}
 AppVerName={#AppName} {#AppVersion}
-AppPublisher={#AppPublisher}
+
+AppPublisher=(c){#StartYearCopyright}-{#CurrentYear} {#AppPublisher}
```

Comment:
Please use the official copyright symbol `©` instead of `(c)`

Reply:
OK